doc/cephfs/mount-using-kernel-driver.rst

```diff
@@ -111,15 +112,14 @@ Persistent Mounts
 To mount CephFS in your file systems table as a kernel driver, add the
 following to ``/etc/fstab``::
 
-    [{ipaddress}:{port}]:/ {mount}/{mountpoint} ceph [name=username,secret=secretkey|secretfile=/path/to/secretfile],[{mount.options}]
+    {name}@{fs_name}=/ {mount}/{mountpoint} ceph [mon_addr={ipaddress},secret=secretkey|secretfile=/path/to/secretfile],[{mount.options}]
```

Comment:
```suggestion
    {name}@{fs_name}=/ {mount}/{mountpoint} ceph [mon_addr={ipaddresses},secret=secretkey|secretfile=/path/to/secretfile],[{mount.options}]
```

Reply:
Technically, if you don't have any mount options you need to put "defaults" in the options field.

This example also leaves out the last two fields. For ceph mounts you'd typically set those to 0's.